ble/BLEProtocol.h

```diff
@@ -44,6 +44,67 @@ namespace BLEProtocol {
+            HUMAN_INTERFACE_DEVICE_HID                     = 960,   /**< Human Interface Device (HID). */
+            KEYBOARD                                       = 961,   /**< Keyboard. */
+            MOUSE                                          = 962,   /**< Mouse. */
+            JOYSTICK                                       = 963,   /**< Joystick. */
+            GAMEPAD                                        = 964,   /**< Gamepad. */
+            DIGITIZER_TABLET                               = 965,   /**< Digitizer Tablet. */
+            CARD_READER                                    = 966,   /**< Card Reader. */
+            DIGITAL_PEN                                    = 967,   /**< Digital Pen. */
+            BARCODE_SCANNER                                = 968,   /**< Barcode Scanner. */
+            GENERIC_GLUCOSE_METER                          = 1024,  /**< Generic Glucose Meter. */
+            GENERIC_RUNNING_WALKING_SENSOR                 = 1088,  /**< Generic Running/Walking Sensor. */
+            RUNNING_WALKING_SENSOR_IN_SHOE                 = 1089,  /**< In Shoe Running/Walking Sensor. */
+            RUNNING_WALKING_SENSOR_ON_SHOE                 = 1090,  /**< On Shoe Running/Walking Sensor. */
+            RUNNING_WALKING_SENSOR_ON_HIP                  = 1091,  /**< On Hip Running/Walking Sensor. */
+            GENERIC_CYCLING                                = 1152,  /**< Generic Cycling. */
+            CYCLING_CYCLING_COMPUTER                       = 1153,  /**< Cycling Computer. */
+            CYCLING_SPEED_SENSOR                           = 1154,  /**< Cycling Speed Sensor. */
+            CYCLING_CADENCE_SENSOR                         = 1155,  /**< Cycling Cadence Sensor. */
+            CYCLING_POWER_SENSOR                           = 1156,  /**< Cycling Power Sensor. */
+            CYCLING_SPEED_AND_CADENCE_SENSOR               = 1157,  /**< Cycling Speed and Cadence Sensor. */
+            PULSE_OXIMETER_GENERIC                         = 3136,  /**< Generic Pulse Oximeter. */
+            PULSE_OXIMETER_FINGERTIP                       = 3137,  /**< Fingertip Pulse Oximeter. */
+            PULSE_OXIMETER_WRIST_WORN                      = 3138,  /**< Wrist Worn Pulse Oximeter. */
+            OUTDOOR_GENERIC                                = 5184,  /**< Generic Outdoor. */
+            OUTDOOR_LOCATION_DISPLAY_DEVICE                = 5185,  /**< Outdoor Location Display Device. */
+            OUTDOOR_LOCATION_AND_NAVIGATION_DISPLAY_DEVICE = 5186,  /**< Outdoor Location and Navigation Display Device. */
+            OUTDOOR_LOCATION_POD                           = 5187,  /**< Outdoor Location Pod. */
+            OUTDOOR_LOCATION_AND_NAVIGATION_POD            = 5188   /**< Outdoor Location and Navigation Pod. */
+        };
+    };
+    typedef AppearanceType::Type AppearanceType_t; /**< Alias for AppearanceType::Type */
```

Comment:
use a @ref in the documentation so that doxygen automatically links the two concepts. `Alias for @ref AppearanceType::Type`